ui/src/containers/ResumeQueueDialog.jsx

```diff
@@ -1,54 +1,33 @@
-/* eslint-disable react/jsx-handler-names */
 import React from 'react';
-import { connect } from 'react-redux';
-import { bindActionCreators } from 'redux';
+import { useDispatch, useSelector } from 'react-redux';
 import { Modal, Button } from 'react-bootstrap';
 import { showResumeQueueDialog } from '../actions/queueGUI';
 
-export class ResumeQueueDialog extends React.Component {
-  constructor(props) {
-    super(props);
-    this.accept = this.accept.bind(this);
-    this.reject = this.reject.bind(this);
-  }
+function ResumeQueueDialog() {
+  const dispatch = useDispatch();
+  const show = useSelector((state) => state.queueGUI.showResumeQueueDialog);
 
-  accept() {
-    this.props.hide();
-  }
-
-  reject() {
-    this.props.hide();
-  }
-
-  render() {
-    return (
-      <Modal show={this.props.show} onHide={this.props.hide}>
-        <Modal.Header>
-          <Modal.Title>Resume Queue</Modal.Title>
-        </Modal.Header>
-        <Modal.Body>
-          Ooops ! The application was closed or there was problems with the
-          connection while the queue was running. Just press Run Queue (found
-          above the queue) if you like to continue execution.
-        </Modal.Body>
-        <Modal.Footer>
-          <Button onClick={this.reject}> OK </Button>
-        </Modal.Footer>
-      </Modal>
-    );
-  }
-}
-
-function mapStateToProps(state) {
-  return {
-    show: state.queue.showResumeQueueDialog,
```

Comment:
Bug here: this should be `queueGUI`. The `ResumeQueueDialog` was likely never showing up.